src/lua/functions/creatures/player/player_functions.cpp

```diff
@@ -1015,7 +1015,7 @@ int PlayerFunctions::luaPlayerSetOfflineTrainingSkill(lua_State* L) {
 	// player:setOfflineTrainingSkill(skillId)
 	Player* player = getUserdata<Player>(L, 1);
 	if (player) {
-		uint32_t skillId = getNumber<uint32_t>(L, 2);
+		int32_t skillId = getNumber<int32_t>(L, 2);
```

Comment:
I realized that in protocolgame the skill is uint, so I think the correct thing is that we use uint here as well.